pip_review/__main__.py

```diff
@@ -49,24 +50,76 @@ def check_output(*args, **kwargs):
 '''
 
 # parameters that pip list supports but not pip install
-LIST_ONLY = set('l local path format not-required exclude-editable include-editable'.split())
+LIST_ONLY = {
+    'l',
+    'local',
+    'path',
+    'format',
+    'not-required',
+    'exclude-editable',
+    'include-editable',
+}
```

Comment:
You wrote exactly the same set with a different notation. I suppose the notation you chose is more canonical or perhaps "cleaner". But does that justify introducing another change in the git history, which future developers might need to search with a `git blame` or a `git bisect` in order to trace why the contents of this set are the way they are?